go/vt/tabletserver/query_splitter.go

```diff
@@ -281,3 +306,27 @@ func (qs *QuerySplitter) parseFloat(pkMinMax *mproto.QueryResult) ([]sqltypes.Va
 	}
 	return boundaries, nil
 }
+
+// TODO(shengzhe): support split based on min, max from the string column.
+func (qs *QuerySplitter) splitBoundariesStringColumn() ([]sqltypes.Value, error) {
+	firstRow := int64(0x0)
+	lastRow := int64(0xFFFFFFFF)
```

Comment:
Assuming key is uniformly distributed, we could category keys into groups using their significant bits. If we use the first four bytes, it could divide keys into upper to 2^32 groups, which means this allows us to run MR using upper to 2^32 = 4,294,967,296 mappers.